src/sinks/file/streaming_sink/mod.rs

```diff
@@ -0,0 +1,13 @@
+use crate::Event;
+use async_trait::async_trait;
+use futures::Stream;
+
+pub mod compat;
+
+#[async_trait]
+pub trait StreamingSink: Send + Sync + 'static {
+    async fn run<'a>(
```

Comment:
no need for `'a`

Reply:
See https://github.com/timberio/vector/pull/1988#discussion_r390264118

Reply:
Make `impl Stream + Send + Sync + 'static` then.

Reply:
Right! :) Applied this change.